iOS/MSMapsModules/MSMapsModules.xcodeproj/xcuserdata/admin.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -7,7 +7,12 @@
 		<key>MSMapsModules.xcscheme_^#shared#^_</key>
 		<dict>
 			<key>orderHint</key>
-			<integer>3</integer>
+			<integer>5</integer>
+		</dict>
+		<key>MSMapsModulesTestApp.xcscheme_^#shared#^_</key>
+		<dict>
+			<key>orderHint</key>
+			<integer>4</integer>
```

Comment:
xcuserdata is specific user data and should not be checked in. xcuserdata should be added to .gitignore. I see that you already have xcuserdata in gitignore, but I think that you are missing a slash at the end of xcuserdata.